Jenkinsfile

```diff
@@ -1 +1 @@
-buildPlugin(platforms: ['windows-i686', 'windows-x86_64'])
+buildPlugin()
```

Comment:
tested on android-aarch64, doesn't work

Reply:
You mean compile or usage? The compile becomes fixed.

Reply:
usage, it already compiles

Reply:
Have you a kodi.log, maybe the shader load brings a error message.

Reply:
If yes maybe related with here https://github.com/xbmc/screensaver.asteroids/tree/master/screensaver.asteroids/resources/shaders and I bring in the separated shaders for GL and GLES